### PR TITLE
Add NanoPolishComp

### PR DESCRIPTION
**Pull request by stephanflemming**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Website: https://a-slide.github.io/NanopolishComp/
Github: https://github.com/a-slide/NanopolishComp
(Bio/Ana)conda:  https://anaconda.org/bioconda/nanopolishcomp
Recipe: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/nanopolishcomp/meta.yaml
(My) parameteroverview: https://docs.google.com/spreadsheets/d/1i6CSDO9pPhSdSmYpy-cKUZB2rVgG2MDLp5j9CRQUOsI/edit?usp=sharing

Fix: https://github.com/galaxyproject/tools-iuc/issues/2992
@mmiladi @bgruening

### CODE REVIEW

**Comment by bgruening** [comment]
@stephanflemming please ping me when this is ready.

**Comment by stephanflemming** [comment]
Implemented as a suite. Will add the wrapper for "Freq_meth_calculate" later

Side note:
Conda version is 0.6.11 (https://anaconda.org/bioconda/nanopolishcomp) whereas the latest release is 0.6.2 (https://github.com/a-slide/NanopolishComp/releases).

Can be reviewed @bgruening @mmiladi

**Comment by stephanflemming** [comment]
Not sure why the tests are not successful. Local testing works.

File: tools/nanopolishcomp/eventaligncollapse.xml [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            <option value="eventalign_collapse" selected="true">Eventalign Collapse</option>
+            <option value="index" selected="true">Index</option>
+            <option value="log">Log</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="out_eventalign_collapse" format="tabular" from_work_dir="results/out_eventalign_collapse.tsv" label="${tool.name} on ${on_string}: Eventalign Collapse">
+            <filter>'eventalign_collapse' in out</filter>
+        </data>
+        <data name="out_index" format="txt" from_work_dir="results/out_eventalign_collapse.tsv.idx" label="${tool.name} on ${on_string}: Index">
```
Comment (bgruening) [inline tools/nanopolishcomp/eventaligncollapse.xml:46]: this is also tabular, left?

File: tools/nanopolishcomp/eventaligncollapse.xml [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            -o 'results'
+            -p 'out'
+            -t \${GALAXY_SLOTS:-4}
+            -v
+        ]]></command>
+    <inputs>
+        <param argument="-i" type="data" format="tabular" label="Select nanopolish eventalign file"/>
+        <param argument="-s" type="boolean" truevalue="-s" falsevalue="" label="Should raw samples be written?" help="If nanopolish eventalign was ran with --samples option"/>
+        <param argument="-r" type="integer" value="0" min="0" label="Set maximum number of reads to parse" help="Use 0 to deactivate this option."/>
+        <param argument="-f" type="select" multiple="true" label="Select statistical fields to compute" help="If nanopolish eventalign was ran with --sample option."> <!-- todo -->
```
Comment (bgruening) [inline tools/nanopolishcomp/eventaligncollapse.xml:29]: If nanopolish eventalign was ran with --sample option.  --> You need to run nanopolsh eventablign with.... to make use of this feature.   this way?

File: tools/nanopolishcomp/eventaligncollapse.xml [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            <option value="mean" selected="true">mean</option>
+            <option value="std">std</option>
+            <option value="median" selected="true">median</option>
+            <option value="mad">mad</option>
+            <option value="num_signals" selected="true">num_signals</option>
```
Comment (bgruening) [inline tools/nanopolishcomp/eventaligncollapse.xml:34]: Number of signals

File: tools/nanopolishcomp/eventaligncollapse.xml [hunk 4]
```diff
@@ -8,6 +8,11 @@
     <stdio></stdio>
     <version_command>NanopolishComp --version</version_command>
     <command detect_errors="exit_code"><![CDATA[
+        ## initialize
+        ## requires a minimum of 3 threads
+        threads=\$((3 > \${GALAXY_SLOTS:-3} ? 3 : \${GALAXY_SLOTS:-3})) &&
```
Comment (bgruening) [inline tools/nanopolishcomp/eventaligncollapse.xml:13]: nice!

**Pull request merged**